packages/prerender/src/errors.tsx

```diff
@@ -1,11 +1,12 @@
 import type { DocumentNode } from 'graphql'
 import { print } from 'graphql'
 
-export class PrerenderGqlError {
+export class PrerenderGqlError extends Error {
   message: string
   stack: string
 
   constructor(message: string) {
+    super('GQL error: ' + message)
     this.message = 'GQL error: ' + message
```

Comment:
Is `this.message` needed now that we're inheriting from `Error`?

Reply:
Yeah I stopped at like 5am last night because I realised my brain had turned to scrambled eggs. I'm going to do a pass over the 3 open prs now but yeah exactly the sort of thing I wanted to spot, thanks!